### ISSUE TITLE
elk-iam-roles.json is missing

### ISSUE THREAD

**Issue by shayanelhami** [SCH-CISM/sch-elk-orchestration#6]
It's mentioned in the README but it's not in the cloudformation folder

**Comment by davidski** [comment]
Good catch! I'm no longer using this solution and can't easily replace this. A PR with the minimum necessary permissions is most welcome!
